### PR TITLE
Add small blurb on legacy nature of AsciiDoc.py

### PR DESCRIPTION
**Pull request by MasterOdin**

### CODE REVIEW

**Comment by mojavelinux** [comment]
This looks good. I left one suggested change inline.

File: README.md [hunk 1]
```diff
@@ -11,20 +11,27 @@ articles, books, ebooks, slideshows, web pages, man pages and blogs.
 AsciiDoc files can be translated to many formats including HTML, PDF,
 EPUB, man page.
 
-AsciiDoc is highly configurable: both the AsciiDoc source file syntax
+AsciiDoc.py is a legacy processor for this syntax, handling an older
+specification of AsciiDoc. As such, this will not properly handle the
```
Comment (mojavelinux) [inline README.md:15]: AsciiDoc never had a specification, so I don't think specification is the right word here. I would prefer "rendition" or, if you prefer, "syntax".

Comment (MasterOdin) [reply README.md]: Rendition is a good word and avoids too much repition of the word "syntax".

Changed

Comment (mojavelinux) [reply README.md]: Thanks!

**Review by mojavelinux — APPROVED** [review]

**Pull request merged**